src/app/components/client/PremiumBadge.tsx

```diff
@@ -64,29 +63,14 @@ const PremiumLayout = (props: Props) => {
       />
     </>
   );
-};
-
-export function PremiumButton(props: Props) {
-  return <PremiumLayout {...props} />;
```

Comment:
@codemist It does not matter too much, but since I’m touching this file either way: As mentioned [here](https://github.com/mozilla/blurts-server/pull/3699#discussion_r1386504849) this wrapper function is redundant since it does not handle any additional logic or functionality.